hack/verify-description.sh

```diff
@@ -54,8 +54,8 @@ else
   versioned_api_files="${*}"
 fi
 
-for file in $versioned_api_files; do
-  $genswaggertypedocs -v -s "${file}" -f - || gen_swagger_result=$?
+for file in "${versioned_api_files}"; do
```

Comment:
oh, this one probably can't be quoted because `versioned_api_files` is kinda an array but also kinda not.

Reply:
This is probably why the test was failing as well. Reverted this to just use `${versioned_api_files}`

https://github.com/koalaman/shellcheck/wiki/SC2066

Reply:
this code is bugged, but fixing it leads to more issues currently 👍 

Trying to track that down here FWIW: https://github.com/kubernetes/kubernetes/pull/73061#discussion_r248937104